File: Domain/Models/Evaluation.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Models
{
    public class Evaluation
    {
        public Evaluation()
        {
            Id = Guid.NewGuid();
            Users = new List<User>();
        }

        public Guid Id { get; set; }

        [Range(0, 5, ErrorMessage = "The evaluation must be between 0 and 5.")]
        public double EvaluationValue { get; set; }

        public bool IsDeleted { get; set; } = false;
        public Guid ServiceId { get; set; }
        public Service Service { get; set; }
        public Guid ServiceOwnerId { get; set; }
        public ServiceOwner ServiceOwner { get; set; }
        public ICollection<User> Users { get; set; }
    }
}

File: ScheduledServicesCenterUI/src/models/evaluation.ts
import { Guid } from 'guid-typescript'; 
import { Service } from './service';
import { User } from './user';


export class Evaluation {

    Id: Guid | undefined;
    EvaluationValue: Number | undefined;
    IsDeleted: boolean | undefined;
    ServiceId: Guid | undefined;
    Service: Service | undefined;
    Users: User[] | undefined;
}

